src/commands/Suggestion.js

```diff
@@ -88,6 +86,33 @@ const _ProcessSuggestion = async (msg, guild, locale, suggestionId, reason, Sugg
         ));
     }
 
+    switch (SuggestionStatus) {
+        case 1:
```

Comment:
Indentation doesn't respect the standard of the project.
See:
https://github.com/Marco4413/JackBot/blob/a7739bf723d9a942442cff722c862a6ac549057e/src/commands/Sticky.js#L40-L52